docs/UserGuide.adoc

```diff
@@ -233,11 +233,6 @@ Loans the book with serial number B00041
 * `loan sn/B00201 sn/B02929 sn/B00203` - Coming in v2.0 +
 Loans the books with serial numbers B00201, B02929 and B00203
 
-Upon loaning a book, a printable loan slip in pdf format will be generated.
```

Comment:
I think it is good to leave this here to let the user know what to expect from the entire loan process